src/features/goblins/bank/lib/bankUtils.ts

```diff
@@ -74,6 +75,27 @@ export function canWithdraw({ item, game }: CanWithdrawArgs) {
     return !cropIsPlanted({ item: "Parsnip", game });
   }
 
+  const stoneReady = Object.values(game?.stones).every((stone) =>
+    canMine(stone)
+  );
+  const ironReady = Object.values(game?.iron).every((iron) => canMine(iron));
```

Comment:
Recommend: Move `ironReady` to below `Tunnel Mole` check and `goldReady` to below `Rocky the Mole` check for performance.